apps/portal/js/dashboard.js

```diff
@@ -243,24 +243,24 @@ $(function () {
      * @param page Current page
      * @returns {Array} List of allowe roles
      */
-    var gerUserAllowedViews = function (page) {
+    var getUserAllowedViews = function (page) {
         $('#ds-allowed-view-list').empty();
         var allowedViews = [];
         var pageViews = JSON.parse(JSON.stringify(page.layout.content));
         var viewKeysArray = Object.keys(pageViews);
-        var viewsArrayLength = viewKeysArray.length;
 
-        for (var i = 0; i < viewsArrayLength; i++) {
+        for (var i = 0, viewsArrayLength = viewKeysArray.length; i < viewsArrayLength; i++) {
```

Comment:
Why have we used initialized viewsArrayLength and then checked "i < viewsArrayLength", instead of directly using i < viewKeysArray.length ?